compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -149,10 +157,14 @@ def __init__(
 
         # mypy ... sometimes you just ain't too bright
         self.container_id = container_id  # type: ignore[assignment]
+        self.user_endpoint_config = user_endpoint_config
 
         self._task_group_id: uuid.UUID | None = None  # help mypy out
         self.task_group_id = task_group_id
 
+        self._user_endpoint_config: dict[str, t.Any] | None = None
+        self.user_endpoint_config = user_endpoint_config
```

Comment:
Probably only need one of these assignments

Reply:
Good call. I followed the standard set for `task_group_id`, but now I see that was only to satisfy mypy and isn't relevant for `user_endpoint_config`.